molviewspec/molviewspec/nodes.py

```diff
@@ -387,6 +411,14 @@ class ComponentExpression(BaseModel):
 ColorT = Union[ColorNamesT, str]  # str represents hex colors for now
 
 
+# TODO: Segmentation too?
+class VolumeRepresentationParams(BaseModel):
+    """
+    Representation node, describing how to represent a volume.
+    """
+    # TODO: add slice
+    type: VolumeRepresentationTypeT = Field(description="Representation type, i.e. isosurface or direct_volume")
```

Comment:
I suppose eventually we will have more params here, e.g. isovalue, slice plane. But if this is just WIP, it's fine to add them later.